diff --git a/src/libstore/uds-proxy-store.cc b/src/libstore/uds-proxy-store.cc
index d93514dc7..690a510b7 100644
--- a/src/libstore/uds-proxy-store.cc
+++ b/src/libstore/uds-proxy-store.cc
@@ -1,4 +1,4 @@
-#include "uds-remote-store.hh"
+#include "uds-proxy-store.hh"
 #include "worker-protocol.hh"
 
 #include <sys/types.h>
diff --git a/src/libstore/uds-proxy-store.hh b/src/libstore/uds-proxy-store.hh
index 7223c68fe..4c3925710 100644
--- a/src/libstore/uds-proxy-store.hh
+++ b/src/libstore/uds-proxy-store.hh
@@ -15,7 +15,7 @@ struct UDSProxyStoreConfig : virtual RemoteStoreConfig
     {
     }
 
-    const std::string name() override { return "UDS Daemon Store"; }
+    const std::string name() override { return "UDS Proxy Store"; }
 
     std::string doc() override;
 };
